Jenkinsfile

```diff
@@ -119,8 +119,18 @@ def go_test_linux() {
 }
 
 def is_authorized(name) {
-  def authorized_user = ['VoVAllen', 'BarclayII', 'jermainewang', 'zheng-da', 'mufeili', 'Rhett-Ying', 'isratnisa']
-  return (name in authorized_user)
+  def devs = ['dgl-bot', 'noreply', 'Rhett-Ying', 'BarclayII', 'jermainewang',
```

Comment:
noreply?

Reply:
if a PR is merged into master, CI will be triggered. the author is `noreply`...

Reply:
I was wrong. In this case, `env.CHANGE_AUTHOR` is null. If it's `null`, let's regard it authenticated?

Reply:
Sure. I'm not familiar. Just let you know that master branch is protected, i.e., only authorized users are allowed to merge to it.